package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-drag-scroll",
-  "version": "1.7.1",
+  "version": "1.7.2",
```

Comment:
It would be useful to do the similar change in package-lock.json for version as well with every release - to be consistent

aliter: 

```console
npm i
```
will also do the same thing just to be sure. FYI .